### DISCUSSION TITLE
Server depreciated: betweentheborders.com Please migrate!

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#535] [General]
The server at betweentheborders.com and its associated TCPServer have been depreciated. All pertinent services have been taken over by reticulum.betweentheborders.com

### What you need to do:

If you're connected to betweentheborders.com, change the entry to reticulum.betweentheborders.com and restart your Reticulum instance.

OLD:
```
# TCP/IP interface to the BetweenTheBorders Hub (community-provided)
  [[RNS Testnet BetweenTheBorders]]
    type = TCPClientInterface
    enabled = yes
    target_host = betweentheborders.com
    target_port = 4242
```

NEW:
```
# TCP/IP interface to the BetweenTheBorders Hub (community-provided)
  [[RNS Testnet BetweenTheBorders]]
    type = TCPClientInterface
    enabled = yes
    target_host = reticulum.betweentheborders.com
    target_port = 4242
```

If you're not connected to Between The Borders over TCP, no action needs to be taken.

If you're connected to multiple TCP endpoints (and you should be) then you may delay cycling your rnsd.

We apologize for the inconvenience and lack of notice.

**Comment by kc1awv** [comment]
Thanks for the heads up!

(I needed an excuse to cycle the hub here, too.)
